src/claimDetail.js

```diff
@@ -59,83 +85,811 @@ function extractData() {
+      sentTo: parsedNote[0],
+    };
+
+    if (note.includes("Error while processing")) {
+      data.type = "ERR";
+    } else if (note.includes("Private Plan Operations")) {
+      data.type = "PRIV";
+    } else if (note.includes("Disability During Unemployment")) {
+      data.type = "DDU";
+    } else if (note.includes("invalid because you did not meet the wage")) {
+      data.type = "INVAL_WAGE";
+    } else if (
+      note.includes("ineligible because you failed") &&
+      note.includes("Form C10")
+    ) {
+      data.type = "INEL_C10";
+    } else if (
+      note.includes("ineligible because you failed") &&
+      note.includes("Form M10")
+    ) {
+      data.type = "INEL_M10";
+    } else if (
+      note.includes("ineligible because you failed") &&
+      note.includes("additional information and medical certification")
+    ) {
+      data.type = "INEL_5107";
+    } else if (note.includes("A request for information has been mailed")) {
+      data.type = "SENT_REQ";
+    } else if (note.includes("No decision has been made")) {
+      data.type = "14_DAY";
+    } else if (
+      note.includes("Request to Claimant for Information was sent to")
+    ) {
+      data.type = "SENT_C10";
+    } else if (note.includes("Request for Medical Information was sent to")) {
+      data.type = "SENT_Mxx";
+    } else if (note.includes("Request for Wage information was sent to")) {
+      data.type = "SENT_Exx";
+    } else if (
+      note.includes(
+        "Notice of Required Pursuit of Workers' Compensation Claim was sent to"
+      )
+    ) {
+      data.type = "SENT_W10";
+    } else if (
+      note.includes("Request to Claimant for Information was generated")
+    ) {
+      data.type = "GEN_C01";
+    } else if (note.includes("Form W01")) {
+      data.type = "GEN_W01";
+    } else if (note.includes("Form M-01")) {
+      data.type = "GEN_M01";
+    } else if (note.includes("An Employer Statement")) {
+      data.type = "GEN_E01";
+    } else {
+      data.type = note;
+    }
+
+    return data;
+  });
+}
+
+function getStepsList(contentList) {
+  return html`<div style="margin-top: 12px">
+    <h3 style="font-size: 16px; line-height: 24px; margin: 0">
+      Steps to complete
+    </h3>
+    <ul
+      style="padding-inline-start: 25px; margin-block-start: 0.5em; margin-block-end: 0.5em"
+    >
+      ${contentList.map((item) => html`<li>${item}</li>`).join("")}
+    </ul>
+  </div>`;
+}
+
+function getStatusContent(
+  newStatus,
+  claimNotes,
+  claimType,
+  receivedDate,
+  nextPayDate,
+  weeklyBenefitRate,
+  balanceRemaining,
+  claimEndDate,
+  lastDayPaid
+) {
+  if (newStatus === "In progress") {
+    return getStepsList(["There's no action for you to take."]);
+  } else if (newStatus === "Information needed") {
+    return claimNotes
+      .map((note, idx) => {
+        let title = "";
+        let body = "";
+
+        const onlineResponseDate = new Date(receivedDate);
```

Comment:
Dust: special compliment here that reading through the content and seeing things like "respond by this date" or "if you havent received by this date" feels so much better and I'm glad we did this :)